apps/backend/src/schemas/system_metrics.py

```
@@ -75,7 +75,7 @@ def validate_disk_available(cls, v, info):
 class SystemMetricCreate(SystemMetricBase):
     """Schema for creating system metrics"""
 
-    time: datetime = Field(default_factory=lambda: datetime.now(datetime.UTC), description="Metric timestamp")
+    time: datetime = Field(default_factory=lambda: datetime.now(timezone.utc), description="Metric timestamp")
```

Ruff (0.12.2)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)